src/main/java/io/leonis/subra/game/formations/PositionFormation.java

```diff
@@ -18,11 +18,11 @@
  */
 @Value
 @Slf4j
```

Comment:
Unrelated to the PR, but do you know why we left `@Slf4j` here?

Reply:
don't know why it was there (perhaps debugging) but I removed it